zeitschrift-fur-kunstgeschichte.csl

```diff
@@ -204,12 +237,25 @@
           </group>
         </else>
       </choose>
+      <choose>
+        <if variable="URL" match="any">
+          <group>
+            <text variable="URL" prefix=", "/>
+            <text term="accessed" prefix=" ("/>
+            <date variable="accessed" prefix=" " delimiter="." suffix=")">
+              <date-part name="day"/>
+              <date-part name="month" form="numeric"/>
+              <date-part name="year"/>
+            </date>
+          </group>
+        </if>
+      </choose>
     </layout>
   </citation>
   <bibliography et-al-min="3" et-al-use-first="1" subsequent-author-substitute="&#8212;&#8212;&#8212;" entry-spacing="0" hanging-indent="true">
     <sort>
-      <key macro="author"/>
-      <key variable="issued"/>
+      <key macro="contributors-long" names-min="3" names-use-first="3"/>
```

Comment:
The copied code would produce something like `author1, author2, author3 et al.` which is wrong here for this journal.

Since `et al.` is standing for "and others" it is normally used only to abbreviate more than one person. Thus, there is normally at least a difference of 2 between these two values.

Yes, please delete those two attributes again. We already have the correct values in the bibliography node above.